C-Plus-Plus/search/Sqrt_Monotonic_Binary_Search.cpp

```diff
@@ -0,0 +1,76 @@
+// In this algorithm we will find the square root of a number `n` upto `p` digits after decimal using the binary search algorithm.
```

Comment:
Few changes: 

- Add Multi-Line Comments 
- Add a beginner-friendly description at the top